File: NodeJS/Unirest.js
var unirest = require('unirest');

var token = '56a351796b80339c09ff1b76';
var requestBody = {
  'collector': token,
  'from': {
    'name': 'Ezequias Dinella',
    'email': 'ezequias@binds.co'
  }
};

var req = unirest('POST', 'https://app.binds.co/api/seeds');

req.headers({'content-type': 'application/json; charset=UTF-8'});
req.send(JSON.stringify(requestBody));

req.end(function(res) {
  if (res.error) {
    throw new Error(res.error);
  }
  console.log(res.body);
});

File: Java/Unirest.java
HttpResponse<String> response = Unirest.post("https://app.binds.co/api/seeds")
  .header("content-type", "application/json; charset=UTF-8")
  .body("{\"collector\":\"56a351796b80339c09ff1b76\",\"from\":{\"name\":\"Charlie Parker\",\"email\":\"hello@binds.co\"}}")
  .asString();
